src/style.css

```diff
@@ -0,0 +1,79 @@
+body {
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  color: white;
```

Comment:
html

Reply:
see the checklist.md
"-font family is usually written for body tag and inherited by others

Reply:
moved to html